cmd/soroban-rpc/internal/methods/get_transactions_test.go

```diff
@@ -239,7 +243,7 @@ func TestGetTransactions_LedgerNotFound(t *testing.T) {
 	}
 
 	response, err := handler.getTransactionsByLedgerSequence(context.TODO(), request)
-	expectedErr := errors.Errorf("[%d] ledger close meta not found: 2", jrpc2.InvalidParams)
+	expectedErr := fmt.Errorf("[%d] ledger close meta not found: 2", jrpc2.InvalidParams)
```

Comment:
Why moving from errors package to fmt?

Reply:
Because our internal errors package uses an outdated version of error wrapping https://github.com/pkg/errors

Reply:
We should remove the package from stellar/go

CC @tamirms @mollykarcher